### ISSUE TITLE
[Windows] Git Delta not working as custom pager

### ISSUE THREAD

**Issue by jeeeem** [jesseduffield/lazygit#2300]
**Describe the bug**
Im trying to use the pager `delta` with this config
``` yml
git: 
  paging:
    useConfig: false
    colorArg: always
    pager: delta -s --dark --paging=never
```
but its not working, 

**Expected behavior**
Should have the same behavior when using `git diff` in the terminal

**Screenshots**
`git diff`
![image](https://user-images.githubusercontent.com/51053482/204258786-c671cc5b-648d-4c03-bccf-5531e8509f06.png)

`lazygit`
![image](https://user-images.githubusercontent.com/51053482/204258983-8d160c83-d0c5-4df4-bfea-f3b25fba029d.png)

**Version info:**
- Lazygit Version
`commit=d1a8b05401ccda1e52215203139a252eb3be3b79, build date=2022-11-14T09:22:31Z, build source=binaryRelease, version=0.36.0, os=windows, arch=amd64, git version=2.38.1.windows.1`

- Git Version
`git version 2.38.1.windows.1`

**Comment by zeevro** [comment]
Isn't this a duplicate of #1453 ?

**Comment by GitMurf** [comment]
@jesseduffield to help you cleanup a bit of the noise, I believe this issue can be closed as a DUP of #1453

**Comment by jesseduffield** [comment]
Indeed, thanks
